charts/dotnet-core/Chart.yaml

```diff
@@ -4,6 +4,6 @@ name: charts-dotnet-core
 type: application
 dependencies:
 - name: charts-core
-  version: 2.0.12
+  version: 2.1.0
```

Comment:
should I update the version in `app-reverser-proxy`, `cron-job`, `job`, `pact-broker` as well?

Reply:
ah okay, so I first need to update the core charts then open another PR to update all depending charts? Will revert the change here then

Reply:
reverted